Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM alpine
```

Comment:
I was building on top of `nixos/nix` image ([Dockerfile](https://github.com/NixOS/nix/blob/master/misc/docker/Dockerfile)) and extending it with our stuff.

In final version (later today) there wont be any Dockerfile at the root of the repo, but there will be a command to rebuild (and push) base image.